lib/cdo/lesson_import_helper.rb

```diff
@@ -75,19 +75,13 @@ def self.create_lesson_resources(cb_resources, course_version_id)
   end
 
   def self.create_activity_sections(activity_markdown, lesson_activity_id, levels)
-    # Find any special syntax, such as tips or remarks, and gather them.
-    # TODO tips and remarks both show up in tip_matches. We filter out remarks
-    # but we should try to do something a bit smarter here.
+    activity_markdown.gsub!(/slide!!!slide-\d+(?:<!-- place where you'd like the icon -->)?/, "[slide]")
```

Comment:
In this line and the one below, the markdown is searched for "easy" replacements. This line looks for the old slide syntax and replaces it with our new `[slide]` syntax. The line below looks for `[guide]` syntax, which we're not using anymore, and removes it.

I don't know why I didn't do this for `[guide]` before now to be honest but oh well.